ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml

```diff
@@ -173,6 +173,17 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       TEST_ARGS: --from-repository quay.io/multi-arch/community-e2e-images
     workflow: openshift-e2e-aws-serial
+- as: ocp-e2e-aws-arm64-single-node
+  interval: 48h
+  steps:
+    cluster_profile: aws-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: arm-ci.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.2xlarge
```

Comment:
On x86 they've been using m5d, which has attached disk storage.  The equivalent class would be m6gd.